rxbinding-support-recyclerview/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.jakewharton.rxbinding.support.v7"
```

Comment:
Do you actually need a package name here? Gradle will append ".test" to the main one automatically.